### PR TITLE
Add craft for stone wall

### PR DESCRIPTION
**Pull request by Imperator-Shlepa**
## About the PR
I added craft for stone wall. 
Я добавил крафт для каменной стены.

## Why / Balance
Players will be able to build their own houses and other buildings. 
Игроки смогут строить себе дома и прочие здания.

## Media

https://github.com/crystallpunk-14/crystall-punk-14/assets/155736295/e0b96f04-621c-4faa-9b58-94e09beb37b1

### CODE REVIEW

**Comment by TheShuEd** [comment]
Rename PR :trollface:

File: Resources/Prototypes/_CP14/Entities/Structures/Walls/walls.yml [hunk 1]
```diff
@@ -55,6 +58,9 @@
     sprite: _CP14/Structures/Walls/bricks_wall.rsi
   - type: IconSmooth
     base: bricksA
+#  - type: Construction
```
Comment (TheShuEd) [inline Resources/Prototypes/_CP14/Entities/Structures/Walls/walls.yml:61]: don't add non-working prototypes even with comments

File: Resources/Prototypes/_CP14/Recipes/Construction/Graphs/Walls/WallBrick.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#- type: constructionGraph
```
Comment (TheShuEd) [inline Resources/Prototypes/_CP14/Recipes/Construction/Graphs/Walls/WallBrick.yml:1]: don't add non-working prototypes even with comments

**Review by TheShuEd — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_CP14/Recipes/Construction/Graphs/Walls/WallBrick.yml (hunk 2 above)
Comment (Imperator-Shlepa) [reply Resources/Prototypes/_CP14/Recipes/Construction/Graphs/Walls/WallBrick.yml:1]: You can close this PR.
I want make new PR with this update and fixed prototypes.

Comment (TheShuEd) [reply Resources/Prototypes/_CP14/Recipes/Construction/Graphs/Walls/WallBrick.yml:1]: ok, if you want

**Pull request closed without merging**